src/character.cpp

```diff
@@ -5226,6 +5226,12 @@ hp_part Character::body_window( const std::string &menu_header,
         } else if( limb_is_broken ) {
             desc += colorize( _( "It is broken.  It needs a splint or surgical attention." ), c_red ) + "\n";
             hp_str = "==%==";
+        } else if( has_trait( trait_NOPAIN ) ) {
+            if( current_hp < maximal_hp / 2 ) {
+                hp_str = colorize( string_format( " %s", _( "Bad" ) ), c_red );
```

Comment:
Does this really need to go via `string_format`? It just adds a space in front of the word. A space that is not even needed. The other branches don't add a space in front of the string. And below where `hp_str` is actually used, a space is added anyway. So why add a space here?
```suggestion
                hp_str = colorize( _( "Bad" ), c_red );
```